solr/solr-ref-guide/src/major-changes-in-solr-8.adoc

```diff
@@ -283,9 +283,9 @@ The login screen's purpose is cosmetic only - Admin UI-triggered Solr requests w
 
 *Distributed Requests*
 
-* The `shards` parameter, used to manually select the shards and replicas that receive distributed requests, now checks nodes against a whitelist of acceptable values for security reasons.
+* The `shards` parameter, used to manually select the shards and replicas that receive distributed requests, now checks nodes against an allow-list of acceptable values for security reasons.
 +
-In SolrCloud mode this whitelist is automatically configured to contain all live nodes.  In standalone mode the whitelist is empty by default.  Upgrading users who use the `shards` parameter in standalone mode can correct this value by setting the `shardsWhitelist` property in any `shardHandler` configurations in their `solrconfig.xml` file.
+In SolrCloud mode this allow-list is automatically configured to contain all live nodes.  In standalone mode the allow-list is empty by default.  Upgrading users who use the `shards` parameter in standalone mode can correct this value by setting the `allowUrls` property in their `solrconfig.xml` file.
```

Comment:
You are going back in time and changing upgrade notes for Solr 8 where there was a `shardsWhitelist` and saying it is `allowUrls` when this isn't the case for 8.x.  I think the terminology change is okay but don't change the particular setting name or that it goes on shardHandler which is true for 8.x.

Reply:
Nice catch! Abusive find & replace.

Reply:
I'll also add a sentence in major-changes-in-solr-9.